java/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry Lambda Java
+
+Layers for running Java applications on AWS Lambda with OpenTelemetry.
+
+## Provided layers
+
+Two types of layers are provided
+
+### Java agent
+
+The [AWS OpenTelemetry Java Agent](https://github.com/aws-observability/aws-otel-java-instrumentation)
```

Comment:
what do we need to do for this to work with the standard OpenTelemetry Java agent? would be better not to depend on vendor-specific distro here if possible

Reply:
Ah that's true - just need to add the aws resource extension, which I guess we said we're OK with doing. Either way I'll change this to the normal agent

Reply:
Fixed